web/src/locales/da.json

```
@@ -86,7 +86,7 @@
     "lowcarbon": "Lav CO₂",
     "renewable": "Vedvarende",
     "electricityproduction": "Produktion",
-    "electricityconsumption": "Forbrug",
+    "electricityconsumption": "El",
```

I know my danish is not great but should it not be: "Elektricitet"

```suggestion
    "electricityconsumption": "Elektricitet",
```

---

Just had a quick chat with Mathias, he recommends either "Elektricitet" or "Strøm". Let's go with the first and update it later in case the other is more appropriate. 🙌

---

I think elektricitet works best as that is the one we use in the related languages (mainly Swedish and German).